tests/helper/helper_generic.go

```diff
@@ -167,3 +168,13 @@ func GetUserHomeDir() string {
 	Expect(err).NotTo(HaveOccurred())
 	return homeDir
 }
+
+// LocalKubeconfigSet checks for the running CI and sets the KUBECONFIG to the temporary config directory respectively
+func LocalKubeconfigSet(context string) {
+	homeDir := GetUserHomeDir()
+	if os.Getenv("CI") == "openshift" {
+		copyKubeConfigFile(filepath.Join("/tmp", "admin.kubeconfig"), filepath.Join(context, "config"))
```

Comment:
This doesn't look right at all. You should not use admin.kubeconfig.
Use the `$KUBECONFIG` if it is set. the same for Kubernetes